### PR TITLE
[MODIFY] Filter DO

### PR DESCRIPTION
**Pull request by watcharaporn-git**
@ps-tubtim

### CODE REVIEW

**Comment by kittiu** [comment]
Please explain in description what you want to do.
Also i think the subject should be,
[IMP] module name, short explain

File: custom-addons/esb/packing_list/models/packing_list.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ class ESBPackingList(models.Model):
     picking_ids = fields.Many2many(
         comodel_name="stock.picking",
         string="Packing List",
-        domain="[('partner_id', '=', partner_id)]",
+        domain="[('partner_id', '=',partner_id),('picking_type_id','!=', company_id)]",
```
Comment (kittiu) [inline custom-addons/esb/packing_list/models/packing_list.py]: Why compare picking type with company?

**Comment by newtratip** [comment]
Hello @watcharaporn-git,

This PR for fix this issue ? http://support.ecosoft.co.th/web#id=836&action=167&model=project.task&view_type=form&menu_id=127

Is this done ?

File: custom-addons/esb/packing_list/models/packing_list.py (hunk 1 above)
Comment (newtratip) [inline custom-addons/esb/packing_list/models/packing_list.py]:
```suggestion
        domain="[('partner_id', '=', partner_id), ('picking_type_id.code', '=', 'outgoing')]",
```

**Review by newtratip — COMMENTED** [review]
@watcharaporn-git  Please fix.

**Review by newtratip — APPROVED** [review]
LGTM :+1:

**Pull request merged**